lucene/analysis/kuromoji/build.xml

```diff
@@ -136,8 +136,8 @@
      </compile>
   </target>
 
-  <target name="test-tools" depends="compile-tools-tests">
-    <test-macro dataDir="src/tools/test" junit.classpath="tools.test.classpath"/>
+  <target name="test-tools" depends="install-junit4-taskdef, compile-tools-tests">
+    <test-macro testsDir="${build.dir}/classes/tools-test" workDir="src/tools/test" junit.classpath="tools.test.classpath"/>
```

Comment:
just an idea for the future: if we are fixing this tooling (e.g. replacing bogus asserts with real checks so it is usable like this PR does), maybe we should simply move all the `tools*` stuff and tests into the regular source tree?

Reply:
yeah, this felt kind of like a dusty corner where things would easily get lost. I'm just learning about the tools and modules concepts in lucene builds. It seems fine to have the builder available in the analysis jar.